mod.yaml

```diff
@@ -3,12 +3,12 @@
 Metadata:
 	Title: Over Powered Mod
 	Description: Crazy modifications to Red Alert by Boolbada
-	Version: 007-Playtest-20170304
+	Version: {DEV_VERSION}
 	Author: Boolbada
 
 RequiresMods:
-	#modchooser: playtest-20170304
 	modchooser: {DEV_VERSION}
+	as: {DEV_VERSION}
```

Comment:
the idea is to include a minimal mod.yaml for the AS.dll so it can be used with multiple `mods` directories. Do you have an idea how that could be done without a mod.yaml?